app/javascript/components/TestimonialSelectModal.tsx

```diff
@@ -110,45 +111,43 @@ export const TestimonialSelectModal = ({
         </>
       }
     >
-      <div>
-        {isLoading && state.reviews.length === 0 ? (
-          <div className="flex items-center justify-center">
-            <LoadingSpinner className="size-8" />
+      {isLoading && state.reviews.length === 0 ? (
+        <div className="flex items-center justify-center">
+          <LoadingSpinner className="size-8" />
+        </div>
+      ) : !isLoading && state.reviews.length === 0 ? (
+        <p>No reviews with text or video yet.</p>
+      ) : (
+        <div className="flex min-h-0 flex-col gap-2">
+          <div className="flex flex-row items-center gap-2">
+            <input
+              type="checkbox"
+              role="checkbox"
+              checked={selectedReviewIds.length === state.reviews.length && state.reviews.length > 0}
+              onChange={toggleSelectAll}
+              aria-label="Select all reviews"
+            />
+            <p>Select all</p>
           </div>
-        ) : !isLoading && state.reviews.length === 0 ? (
-          <p>No reviews with text or video yet.</p>
-        ) : (
-          <>
-            <div className="flex flex-row items-center gap-2">
-              <input
-                type="checkbox"
-                role="checkbox"
-                checked={selectedReviewIds.length === state.reviews.length && state.reviews.length > 0}
-                onChange={toggleSelectAll}
-                aria-label="Select all reviews"
+          <section className="flex min-h-0 flex-1 flex-col gap-4 overflow-y-auto p-1">
```

Comment:
1. `p-1` so borders are visible clearly inside the scroll container
2. `parapraghs` replaced with `flex flex-col gap-4` to ease `_containers.scss` migration